char dir_command #direction command
---
string msg_feedback

